packages/firestore/src/local/indexeddb_query_cache.ts

```diff
@@ -97,36 +97,75 @@ export class IndexedDbQueryCache implements QueryCache {
     );
   }
 
-  addQueryData(
+  private saveQueryData(
```

Comment:
nit: We usually put private helpers after the public methods that use them.

Reply:
Done.